src/classes/Passage.ts

```diff
@@ -1,203 +1,158 @@
-        }
-
-        if (passageAuthToken) {
-            const userID = await this.validAuthToken(passageAuthToken);
-            if (userID) return userID;
-            else {
-                throw new PassageError('Could not validate auth token. You must catch this error.');
-            }
-        } else {
-            throw new PassageError(
-                "Could not find authentication cookie 'psg_auth_token' token. You must catch this error.",
-            );
-        }
-    }
-
-    /**
-     * Determine if the provided token is valid when compared with its
-     * respective public key.
-     *
-     * @param {string} token Authentication token
-     * @return {string} sub claim if the jwt can be verified, or Error
-     */
-    async validAuthToken(token: string): Promise<string | undefined> {
-        try {
-            const { kid } = decodeProtectedHeader(token);
-            if (!kid) {
-                throw new PassageError('Could not find valid cookie for authentication. You must catch this error.');
-            }
-
-            const {
-                payload: { sub: userID },
-            } = await jwtVerify(token, this.jwks);
-            if (userID) return userID.toString();
-
-            throw new PassageError('Could not verify token identity. You must catch this error.');
-        } catch (e) {
-            if (e instanceof Error)
-                throw new PassageError(`Could not verify token: ${e.toString()}. You must catch this error.`);
-
-            throw new PassageError(`Could not verify token. You must catch this error.`);
-        }
-    }
-
-    /**
-     * Create a Magic Link for your app.
-     *
-     * @param {MagicLinkRequest} magicLinkReq options for creating a MagicLink.
-     * @return {Promise<MagicLinkObject>} Passage MagicLink object
-     */
-    async createMagicLink(magicLinkReq: MagicLinkRequest): Promise<MagicLinkObject> {
-        try {
-            const response = await fetch({
-                body: magicLinkReq,
-                headers: {
-                    Authorization: `Bearer ${this.#apiKey}`,
-                },
-                method: 'POST',
-                url: `https://api.passage.id/v1/apps/${this.appID}/magic-links/`,
-            });
-
-            return response.magic_link;
-        } catch (err) {
-            throw new PassageError('Could not create a magic link for this app.', err as FetchError);
-        }
-    }
-
-    /**
-     * Get App Info about an app
-     *
-     * @return {Promise<AppObject>} Passage App object
-     */
-    async getApp(): Promise<AppObject> {
-        try {
-            const response = await fetch({
-                method: 'GET',
-                url: `https://api.passage.id/v1/apps/${this.appID}`,
-            });
-
-            return response.app;
-        } catch (err) {
-            throw new PassageError('Could not fetch app.', err as FetchError);
-        }
-    }
-}
+// import { Request } from 'express-serve-static-core';
```

Comment:
Commenting out the custom functions for now. This work will come in a follow up PR.